sql/util_time.sql

```diff
@@ -46,3 +57,31 @@ RETURNS INT8 AS '@MODULE_PATHNAME@', 'ts_continuous_agg_watermark_materialized'
 
 CREATE OR REPLACE FUNCTION _timescaledb_functions.subtract_integer_from_now( hypertable_relid REGCLASS, lag INT8 )
 RETURNS INT8 AS '@MODULE_PATHNAME@', 'ts_subtract_integer_from_now' LANGUAGE C STABLE STRICT;
+
+-- Convert integer UNIX timestamps in microsecond to a timestamp range.
+CREATE OR REPLACE FUNCTION _timescaledb_functions.as_multirange(
+    low_usec bigint, high_usec bigint, tstzrange
+) RETURNS TSTZMULTIRANGE AS
+$body$
+  select multirange(tstzrange(_timescaledb_functions.to_timestamp(low_usec),
+			      _timescaledb_functions.to_timestamp(high_usec)));
+$body$ LANGUAGE SQL
+SET search_path TO pg_catalog, pg_temp;
+
+-- Convert integer UNIX timestamps in microsecond to a timestamp range.
+CREATE OR REPLACE FUNCTION _timescaledb_functions.as_multirange(
+    low_usec bigint, high_usec bigint, TSRANGE
+) RETURNS TSMULTIRANGE AS
+$body$
+  select multirange(tsrange(_timescaledb_functions.to_timestamp_without_timezone(low_usec),
+			    _timescaledb_functions.to_timestamp_without_timezone(high_usec)));
+$body$ LANGUAGE SQL
+SET search_path TO pg_catalog, pg_temp;
+
+-- Helper function to construct a range given an existing type from
+-- UNIX timestamps in microsecond precision.
+CREATE OR REPLACE FUNCTION _timescaledb_functions.make_range(
+    base anyrange, low_usec anyelement, high_usec anyelement
```

Comment:
why is the first argument `base` is needed? I don't see it's being used anywhere in `ts_make_range()`.